### PR TITLE
misc martian fixes #2 [NEEDS REVIEW]

### LINKED ISSUE #16476: Martians have a fucky bicon

### LINKED ISSUE #16585: squid martians are immune to explosive maracas.
(WEB REPORT BY: boosterrooster REMOTE: 172.93.109.202:7777)
# Revision
dc1c0dbbcf447f7a9e40003dacc52fe27127821a

# Description
I threw three of the explosive maracas at a one and it hit them directly and it didnt die.
# Steps to Reproduce
Toss a few at a those squid leg freaks.
# What you Expected
Them to die after at least the second one.
# What Actually Happened
They kept holding their own set of maracas and throwing them back at me while I sent into crit after one of them.

### PR DESCRIPTION
**Pull request by MadmanMartian**
**Checklist**
- [x] Squids can now be exploded closes #16585
- [x] Fix squid examine closes #16476
- [ ] Figure out what the fuck is going on in #16497
- [x] Shaking hands with/as a squid should say 'shakes tentacles with'
- [x] Redo UI - ~~Need spriter~~ 
- [x] Investigate as to why you can't rightclick as a squid - **COULD NOT REPRODUCE**
chugmonkey_suggestions
- [x] Investigate into why martians can't use cloners
- [x] Investigate why martians can't push past people
- [x] Tweak martians hypothermic gates so they're not spammed constantly

_**Gallery of unreported bugs**_

jubisloviu_not_reporting_bugs
razz_unreported_bugs
fucking_hypothermia

**_Mandatory anime avatar bully post_**
bullied_by_anime_nerds

:cl:

* rscadd: Martian new HUD courtesy of realestestate

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -274,8 +274,8 @@
 
 					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 					M.visible_message( \
-						"<span class='notice'>[M] shakes hands with [src].</span>", \
-						"<span class='notice'>You shake hands with [src].</span>", \
+						"<span class='notice'>[M] shakes [ismartian(src) ? "tentacles" : "hands"] with [src].</span>", \
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/carbon.dm]: should be checking ismartian(M)

**Comment by N3X15** [comment]
You need to put the icons you added or changed in the description so we can see what the fuck you did.

**Comment by DamianX** [comment]
image

**Comment by DamianX** [comment]
Also martians can hug themselves. F-feature?

**Comment by DamianX** [comment]
#16497 still happens

**Comment by DamianX** [comment]
Should we merge this and look into the handcuffs oddities at a later time?

**Comment by MadmanMartian** [comment]
If you wish

**Pull request merged**